pulsar_metrics/metrics/drift.py

```diff
@@ -87,7 +84,6 @@ def __init__(self, metric_name: str, feature_name: str, **kwargs):
 
         try:
             self._feature_name = feature_name
-            # self._column = data[feature_name]
 
         except Exception as e:
```

Comment:
Minor comment : Please write the Error or exception statement for clear readability ( apply it other places  too)

Reply:
Added more explicit error messages for readability